File: test/handler.py


schema = {
    'required': [
        'name',
        'surname',
    ]
}



async def handle(name, surname):
    return {
        'status': 200,
        'body': {
            'message': f'ciao {name} {surname}!'
        },
        'headers': {
            'Ciaooo': 'adsf.com'
        }
    }

File: test-node/handler.ts



export const schema = {
    type: 'object',
    required: [
        'name'
    ]
}


export const handler = async ({ body: { name }}) => {
    // throw new Error('ho no!')
    return {
        body: {
            message: `ciaoo ${name}`
        },
        status: 200,
    }
}
